src/hooks/JB721Delegate/DelegateVersion.ts

```diff
@@ -0,0 +1,18 @@
+import { useIsJB721DelegateV1 } from './IsJB721DelegateV1'
+import { useIsJB721DelegateV1_1 } from './IsJB721DelegateV1_1'
+import { DV_V1, DV_V1_1 } from 'constants/delegateVersions'
+import { DelegateVersion } from 'models/nftRewardTier'
+
+export function useJB721DelegateVersion({
+  dataSourceAddress,
+}: {
+  dataSourceAddress: string | undefined
+}): DelegateVersion | undefined {
+  if (!dataSourceAddress) return
+
+  if (useIsJB721DelegateV1({ dataSourceAddress })) return DV_V1
+
+  if (useIsJB721DelegateV1_1({ dataSourceAddress })) return DV_V1_1
```

Comment:
**issue** hooks cannot be called conditionally like this. We'll prob have to call them both, and then return the version for the one that returned `true.`